index.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Africa Agriculture Summit|2023</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" />
+  <link rel="stylesheet" href="./css/style.css" />
+  <script src="./js/index.js" defer></script>
+</head>
+
+<body>
+  <header>
+    <div class="desktop-nav">
+      <div class="face-twitt">
+        <img src="./images/facebook.png" alt="facebook" class="facebook">
+        <img src="./images/twitter.png" alt="twitter" class="twitter">
+      </div>
+      <ul>
+        <li><a href="#">English</a></li>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    </div>
+    <img src="./images/menu.png" class="menu" alt="menu-icon">
+    <div class="desktop-items">
+      <a href="index.html"><img src="./images/logo.png" alt="" class="desktop-logo"></a>
+      <ul>
+        <li><a href="about.html" class="about">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Sponsors</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#" class="campaign">AA Campaign</a></li>
+      </ul>
+    </div>
+    <nav class="hidden">
+      <img src="./images/cancel.png" alt="" class="cancel">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Feature</a></li>
+      </ul>
+    </nav>
+    <main class="main-wrapper">
+      <div class="h1-cont">
+        <h1 class="title">Africa Agriculture Summit 2023</h1>
+      </div>
+      <p class="motto">
+        Bringing together farmers, suppliers, processors, brands, academia,
+        conservation organizations and the public sector.
+      </p>
+      <p class="date">23 Dec 2023</p>
+      <p class="venue">Lagos Continental Hotel, Lag, Nig.</p>
+    </main>
+  </header>
+  <main class="">
+    <section class="main-program">
+      <h4 class="main-program-header">Main Program</h4>
+      <div class="h_line"></div>
+      <div class="program-card-section">
+        <div class="main-program-card1">
+          <span class="material-symbols-outlined">
+            mark_unread_chat_alt
+          </span>
+          <p class="main-program-card-header">Sustainability</p>
+          <p class="main-program-card-desp">
+            Listen to welcome introduction by Agbabiaka Abdulquadri,
+            CEO|General Manager, Just Agric Company.
+          </p>
+        </div>
+        <div class="main-program-card1">
```

Comment:
- [ ] Dear @oluwatobi-fasasi, These exercises are meant to make us adopt working with a given design.  This includes color, font, background images, and more. I can see you have applied a green background (this might be right according to your objection but not according to the original design), But for the sec of this project we need to use something similar, something white. 
- [ ] besides that please take note of the font sizes you are using and the alignment of the texts inside your page. Please try to use a different screen to see the changes and make sure it looks identical to the original design before the next review request. 
### Your homepage
![image](https://user-images.githubusercontent.com/69077061/227600880-a6815936-6e2c-4864-ad80-aa6bf29af879.png)
### Original design home page
![image](https://user-images.githubusercontent.com/69077061/227601695-4c859fc5-832d-4f26-8882-34b5a91f82aa.png)